README.md

```diff
@@ -554,6 +554,33 @@ This is the expected result after removing preferences with the URL code specify
 }
 ```
 
+### https://[APPURL]/api/Terminology/[id]/preferred-terminology
+
+#### GET
+
+Return the reference to the preferred-terminology related to the `Terminology` (specified by id).<br>
+Returns the preferred-terminology at the Terminology("self") level. <br> Expected return example below.
+
+```json
+{
+    "references": {
+        "reference_1": "/Terminology/tm--example1",
```

Comment:
Let's break this into two distinct references, one to tm--example1 and the other tm--example2, each as full blown references

"references": [
  { "reference": "Terminology/tm--example1"},
  {"reference": "Terminology/tm--example2"}
]